backend/tournesol/models/entity.py

```diff
@@ -121,6 +121,19 @@ def update_n_ratings(self):
         )
         self.save(update_fields=["rating_n_ratings", "rating_n_contributors"])
 
+    def auto_remove_from_rate_later(self, user):
```

Comment:
I think we can simply call the method `remove_from_rate_later`.

Because in fact, it's as automatic as the metadata refresh method: it runs only when it's called, so it's not really automatic. Even the method docstring suggests it's an activated effect ("When called, the entity...") and not a triggered effect.

The automatic behaviour is made possible by the call of this method by views, so `remove_from_rate_later` feels more appropriate to me.

Reply:
I was trying to find a better name. 

When calling a method called `remove_from_rate_later` I expect the entity is removed from the rate later list, not that it is removed condittionally to having been compared 4 times. Leaving the keyword `auto` make it sound like it doesn't always do the removal.

Any other naming ideas?
- `check_should_remove_rate_later`
- `remove_rate_later_if_enough_comparisons` (<- serious)

Reply:
You're right, `auto` suggests the entity is not always removed.

Maybe this?

- `remove_rl_when_n_comparisons(n_comparisons=4)`

I have no better idea, so I'm fine with the original `auto_remove_from_rate_later`.